specification/servicebus/resource-manager/Microsoft.ServiceBus/stable/2017-04-01/examples/Migrationconfigurations/SBMigrationconfigurationCreateAndStartMigration.json

```diff
@@ -21,7 +21,8 @@
         "properties": {
           "provisioningState": "Accepted",
           "targetNamespace": "/subscriptions/SubscriptionId/resourceGroups/ResourceGroup/providers/Microsoft.ServiceBus/namespaces/sdk-Namespace-4028",
-          "postMigrationName": "sdk-PostMigration-5919"
+          "postMigrationName": "sdk-PostMigration-5919",
+		  "migrationState":"Initiating"
```

Comment:
Nit: fix formatting (remove indentation and add space after colon).  Comment applies to other examples.